src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partnerRole = '';
+
+  if (isMarried) {
+    partnerRole = (sex === 'm') ? 'wife' : 'husband';
+  }
+
+  return (
+    <>
```

Comment:
useless [React.Fragment](https://reactjs.org/docs/fragments.html) here - you have 1 child - section